tools/ncvarsort.py

```diff
@@ -118,16 +126,31 @@ def main():
     # as well as all metadata to the new file.
     for i in range(len(varnames_list_sorted)):
         v_name = varnames_list_sorted[i]
-        varin = dsin.variables[v_name]
-        outVar = dsout.createVariable(v_name, varin.datatype, varin.dimensions)
+        varin  = dsin.variables.get(v_name)
+        v_type = dsin.variables[v_name].typecode()
+        v_dims = varin.dimensions
+        print(" V_NAME = ",v_name,"; V_TYPE = ",v_type,"; V_DIMS = ",v_dims)
```

Comment:
do you want to keep this print statement, or was it just for debugging?

Reply:
It was just for debugging, I will delete it.